Connext_DDS_QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,108 @@
+
+### License [3.iii]
+
+`RTI-Connext-DDS` is governed by the [*Real Time Innovations Software License Agreement*](https://community.rti.com/content/page/rti-software-license-agreement).
+
+### Copyright Statements [3.iv]
+
+The header files provided in the library indicate they are copyrighted by *“Real-Time Innovations, Inc.”*. It is not stated if this is enforced with any kind of linter analysis.
+
+## Testing [4]
+
+### Feature Testing [4.i]
+
+There is a public [blog entry](https://www.rti.com/blog/software-testing-at-rti) from the RTI company summarizing the standards they have for testing, and explaining they have unit testing, feature testing and interoperability testing. The test suite, environment nor the test code is available publicly.
+
+### Public API Testing [4.ii]
+
+There is a public [blog entry](https://www.rti.com/blog/software-testing-at-rti) from the RTI company summarizing the standards they have for testing, and explaining they have unit testing, feature testing and interoperability testing. The test suite, environment nor the test code is available publicly.
+
+### Coverage [4.iii]
+
+There is no public information available regarding the software's code coverage analysis or results.
+
+### Performance [4.iv]
+
+RTI provides performance testing software [here](https://community.rti.com/downloads/rti-connext-dds-performance-test). With this tool, it is possible to test latency and throughput of `RTI-Connext-DDS`. According to this [blog entry](https://www.rti.com/blog/software-testing-at-rti) explaining how RTI manages testing, this tool is used to ensure the software does not regress below a certain percentage when new features are added.
+
+### Linters and Static Analysis [4.v]
+
+There is no public information available about this software's linting or static analysis.
+
+## Dependencies [5]
+
+### Direct/Optional Runtime ROS Dependencies [5.i]/[5.ii]
+
+`RTI-Connext-DDS` has no direct/optional runtime ROS dependencies.
+
+### Direct Runtime non-ROS Dependency [5.iii]
+
+`RTI-Connext-DDS` depends on the external library `openssl1.1.1`. This library comes compiled and bundled with the RTI software. It is expected that this library is tested as a part of the `RTI-Connext-DDS` testing tools environment.
+
+## Platform Support [6]
+
+The `RTI-Connext-DDS` provides support for Linux, MacOs and Windows platforms, as it can be seen in their [supported platforms](https://community.rti.com/static/documentation/connext-dds/5.3.1/doc/manuals/connext_dds/html_files/RTI_ConnextDDS_CoreLibraries_ReleaseNotes/index.htm#ReleaseNotes/System_Requirements.htm#Table_SupportedPlatforms) landing page.  However, there is no declared support for Ubuntu Focal nor MacOs 10.14.
+
+## Security [7]
+
+###  Vulnerability Disclosure Policy [7.i]
+
+`RTI-Connext-DDS` does not have a Vulnerability Disclosure Policy.
```

Comment:
This still surprises me, but I can't find anything myself. Or even reports of security vulnerability disclosures.